### PR TITLE
HP-362 - Documentation HEAL edits

### PR DESCRIPTION
**Pull request by xritter1**
Link to Jira ticket if there is one: https://ctds-planx.atlassian.net/browse/HP-362

### Environments
QA-HEAL

### Description of changes
HEAL-Documentation edits (new screenshots; new/updated sections; etc - see ticket above for full list)

### CODE REVIEW

File: qa-heal.planx-pla.net/dashboard/Public/documentation/index.html [hunk 1]
```diff
@@ -1276,10 +1704,22 @@ <h2 id="Tutorials" class="section-title">6. Tutorial Videos</h2>
...
                         If your Browser does not support watching this video, here's
                         a <a href="videos/Opioid_Ovderdose_Traj_CDCWonder_Demo_2021-07-21.mp4">link to the video</a> instead. <br> <br>
-                    
                     </li>
                 </ul>
-            </div> </div>
+
+                <h2 id="Contact" class="section-title">Contact</h2>
+                <p>
+                    Need help? Please contact our help desk at heal-support@datacommons.io.
```
Comment (williamhaley) [inline qa-heal.planx-pla.net/dashboard/Public/documentation/index.html:1712]: Do we want the email to be a `mailto:` link?

File: qa-heal.planx-pla.net/dashboard/Public/documentation/index.html [hunk 2]
```diff
@@ -1215,6 +1615,34 @@ <h2 id="FAQ" class="section-title">5. FAQ</h2>
...
+                        <br><br>
+                    </li>
+                    <li>
+                        <span style="font-weight: bold;">How do I work with these Tutorial Notebooks?</span><br><br>
+                        Click <a href="#WorkingTutorialNotebooksInteractively">here</a> to 
+                        see a guide of how to work with the Tutorial Notebooks.
+                        <br><br>
+                    </li>
+                    <li>
+                        <span style="font-weight: bold;">How do I link my account to a FAIR enabled repository? </span><br><br>
```
Comment (williamhaley) [inline qa-heal.planx-pla.net/dashboard/Public/documentation/index.html:1635]: No need to change this, but `<br>` for explicit spacing is a bit of a bad smell. It's valid to use a block-level element like `p` inside an `li` if you want to create new-lines. Ideally `bootstrap` or another styling framework helps with simple things like spacing, but this works! 👍

File: qa-heal.planx-pla.net/dashboard/Public/documentation/index.html [hunk 3]
```diff
@@ -913,36 +1235,72 @@ <h4 id="CurrentlyAvailableNotebooks" class="section-title">Currently available N
...
-                    <a href="#LinkingAccessTo">here</a>.
                 </p>
-            
-            <h5 id="HEALNotebookBACPAC" class="section-title">a) HEAL_Workspace_Demo_BACPAC.ipynb (“BACPAC synthetic data analysis”)</h5>
+
                 <ol>
                     <li>
-                        Find and select the study by the project number (1U24AR076730-01) on the <a href="http://healdata.org/discovery">Discovery Page</a>. Project numbers and datasets are given in the previous section.<br><br>
+                        Find and select the study by the project number (in this example: 1U24AR076730-01) 
+                        on the <a href="http://healdata.org/discovery">Discovery Page</a>.
```
Comment (williamhaley) [inline qa-heal.planx-pla.net/dashboard/Public/documentation/index.html:1302]: Very minor detail, but all links should be `https` by default. Just saves a basic http -> https redirect for modern browsers.

**Review by williamhaley — APPROVED** [review]
A couple inconsequential notes, but lgtm!

File: qa-heal.planx-pla.net/dashboard/Public/documentation/index.html (hunk 2 above)
Comment (xritter1) [reply qa-heal.planx-pla.net/dashboard/Public/documentation/index.html:1635]: I believe the whole page moves to another platform soon..

**Pull request merged**